Library/Formula/fstar.rb

```diff
@@ -0,0 +1,92 @@
+class Fstar < Formula
+  desc "Language with a type system for program verification"
+  homepage "https://www.fstar-lang.org/"
+  url "https://github.com/FStarLang/FStar.git",
+    :tag => "v0.9.1.1",
+    :revision => "c41ea17778ff2fef4e75d56604ce1804e1985f33"
+  head "https://github.com/FStarLang/FStar.git"
+
+  depends_on "mono"   => :build
```

Comment:
Don't worry about equal spacing around the `=>`. I heavily prefer out of sync lines over random spacing, personally. One space is fine. `depends_on "mono" => :build` and such.